src/main/java/org/jenkinsci/plugins/ghprb/upstream/GhprbUpstreamStatusListener.java

```diff
@@ -115,7 +116,7 @@ public void onStarted(AbstractBuild<?, ?> build, TaskListener listener) {
 
     // Sets the status to the build result when the job is done, and then calls the createCommitStatus method to send it to GitHub
     @Override
-    public void onCompleted(AbstractBuild<?, ?> build, TaskListener listener) {
+    public void onCompleted(AbstractBuild<?, ?> build, @Nonnull TaskListener listener) {
```

Comment:
The method you're overriding has it defined.  It's better to copy the declaration of the parent method exactly.